supervisor/resolution/checks/core_version.py

```diff
@@ -0,0 +1,59 @@
+"""Helpers to check core version."""
+from enum import Enum
+from pathlib import Path
+from typing import List
+
+from ...const import SUPERVISOR_DATA, CoreState
+from ..const import ContextType, IssueType
+from .base import CheckBase
+
+CC_PATH = Path(SUPERVISOR_DATA, "homeassistant", "custom_components")
+
+
+class VersionReference(str, Enum):
+    """Version references."""
+
+    CUSTOM_COMPONENTS_BELOW_2021_1_3 = "custom_components_below_2021_1_3"
+
+
+class CheckCoreVersion(CheckBase):
+    """CheckCoreVersion class for check."""
+
+    async def run_check(self) -> None:
+        """Run check if not affected by issue."""
+        if not self.sys_homeassistant.version:
+            # Can't determine version.
+            return
+
+        if self.sys_homeassistant.version < "2021.1.3":
```

Comment:
Does this need normalizing

Reply:
It's handled by AwesomeVersion

Reply:
We need should catch the exception and skip the test if they get trow for version check. That makes sure the landing page i.e. don't break it, so we can save the check above

Reply:
on landingpage you do not have any custom components yet